packages/engine/Source/Scene/Model/Model3DTileContent.js

```diff
@@ -263,6 +263,11 @@ Model3DTileContent.prototype.update = function (tileset, frameState) {
         : undefined;
   }
 
+  const tilesetEnvironmentMapManager = tileset.environmentMapManager;
+  if (model.environmentMapManager !== tilesetClippingPlanes) {
+    model._environmentMapManager = tilesetEnvironmentMapManager;
```

Comment:
Should this be using the actual setter so all that logic runs correctly?
```suggestion
    model.environmentMapManager = tilesetEnvironmentMapManager;
```

Reply:
This is intentionally bypassing the setter: The `Cesium3DTileset` is the singular owner across all it's tiles.

Reply:
BTW, this "owner" pattern is used across 3D Tiles and models for several other properties already, including image-base lighting parameters, clipping planes, clipping regions, etc...